src/service/resource.js

```diff
@@ -240,10 +243,21 @@ export class Resource {
    * @return {number}
    */
   getPriority() {
+    if (this.priorityOverride_ != -1) {
```

Comment:
Is this part necessary? Wouldn't the element just return the new value from `BaseElement#getPriority`?